### ISSUE TITLE
Development Roadmap

### ISSUE THREAD

**Issue by beeing** [mikecao/umami#639]
Hi @mikecao, thanks for the great project that you've contributed. It definitely helped a lot of software and website developers.

Just wondering whether is there a roadmap on what you have in mind for the direction of this software.

This will probably help on new users and engage contributors for specific directions.

Thanks and all the best.

**Comment by mikecao** [comment]
There is no specific roadmap for features other than the enhancements and bug fixes on this board. But my goal is to keep the app as small and simple as possible. I don't want it to be bloated like... well Google Analytics.

**Comment by kunago** [comment]
To be honest, it's way better than other "simple" tools. I tried to install one very simple Google Analytics alternative claiming it was very small and efficient. It efficiently crashed my quite powerful VPS 3 times before I was able to discover the cause. Poor IT support...

Excellent job. It was very easy to install and setup, also it is not a resource hog. Browsing around it has all the stats needed without all that GA non-sense. Love it.
